packages/graphql-docs-generator/__tests__/generator/generateAllOperations.test.ts

```diff
@@ -69,111 +69,3 @@ describe('generateAllOperations', () => {
     expect(generateOperation).toHaveBeenCalledWith(mockFields.f1, mockSchema, maxDepth, generateOptions);
   });
 });
-
-describe('Test generating operations by not retaining case style', () => {
-  beforeEach(() => {
-    jest.clearAllMocks();
-    getFields.mockReturnValue({UPPERCASE: 'value'});
-  });
-  const generateOptions: GQLDocsGenOptions = { useExternalFragmentForS3Object: true, retainCaseStyle: false };
-  it('generateQueries - should call generateOperation', () => {
-    expect(generateQueries(operations, mockSchema, maxDepth, generateOptions)).toEqual([
-      {
-        type: 'query',
-        name: 'Uppercase',
-        ...mockOperationResult,
-      },
-    ]);
-    expect(generateOperation).toHaveBeenCalledWith('value', mockSchema, maxDepth, generateOptions);
-    expect(getFields).toHaveBeenCalled();
-    expect(generateOperation).toHaveBeenCalledTimes(1);
-    expect(getFields).toHaveBeenCalledTimes(1);
-  });
-
-  it('generateMutation - should call generateOperation', () => {
-    expect(generateMutations(operations, mockSchema, maxDepth, generateOptions)).toEqual([
-      {
-        type: 'mutation',
-        name: 'Uppercase',
-        ...mockOperationResult,
-      },
-    ]);
-    expect(generateOperation).toHaveBeenCalledWith('value', mockSchema, maxDepth, generateOptions);
-    expect(getFields).toHaveBeenCalled();
-    expect(generateOperation).toHaveBeenCalledTimes(1);
-    expect(getFields).toHaveBeenCalledTimes(1);
-  });
-
-  it('generateSubscription - should call generateOperation', () => {
-    expect(generateSubscriptions(operations, mockSchema, maxDepth, generateOptions)).toEqual([
-      {
-        type: 'subscription',
-        name: 'Uppercase',
-        ...mockOperationResult,
-      },
-    ]);
-    expect(generateOperation).toHaveBeenCalledTimes(1);
-    expect(getFields).toHaveBeenCalledTimes(1);
-    expect(generateOperation).toHaveBeenCalledWith('value', mockSchema, maxDepth, generateOptions);
-  });
-});
-
-describe('test capitalizeFirstLetter', () =>{
```

Comment:
I believe this method is still being used. So we might want to retain this test suite.